.azure/scripts/setup-kubernetes.sh

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/env bash
 set -xe
+set -o errexit
 
 rm -rf ~/.kube
 
 KUBE_VERSION=${KUBE_VERSION:-1.21.0}
 MINIKUBE_REGISTRY_IMAGE=${REGISTRY_IMAGE:-"registry"}
 COPY_DOCKER_LOGIN=${COPY_DOCKER_LOGIN:-"false"}
 
-DEFAULT_MINIKUBE_MEMORY=$(free -m | grep "Mem" | awk '{print $2}')
-DEFAULT_MINIKUBE_CPU=$(awk '$1~/cpu[0-9]/{usage=($2+$4)*100/($2+$4+$5); print $1": "usage"%"}' /proc/stat | wc -l)
+DEFAULT_KIND_MEMORY=$(free -m | grep "Mem" | awk '{print $2}')
+DEFAULT_KIND_CPU=$(awk '$1~/cpu[0-9]/{usage=($2+$4)*100/($2+$4+$5); print $1": "usage"%"}' /proc/stat | wc -l)
 
-MINIKUBE_MEMORY=${MINIKUBE_MEMORY:-$DEFAULT_MINIKUBE_MEMORY}
-MINIKUBE_CPU=${MINIKUBE_CPU:-$DEFAULT_MINIKUBE_CPU}
+KIND_MEMORY=${KIND_MEMORY:-$DEFAULT_KIND_MEMORY}
+KIND_CPU=${KIND_CPU:-$DEFAULT_KIND_CPU}
```

Comment:
I think you shouldn't rename it like that. These options should be common for all clusters available via the script.

Reply:
Yeah, it's just left-over I would name it f.e. `DEFAULT_CLUSTER_MEMORY` etc.